src/index.tsx

```diff
@@ -31,6 +31,7 @@ render(
         appReducer={appReducer}
         deviceSelect={<BoilerplateDeviceSelector />}
         sidePanel={<SidePanel />}
+        feedback={{ categories: ['Board Configurator'] }}
```

Comment:
What's the point of this category?
The feedback will automatically be classified as coming from the board configurator app.